<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>F1D022057 - Lalu Olfata Vedora Zurji</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 40, 40);
</string>
  </property>
  <widget class="QSlider" name="fontSizeSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::sub-page:horizontal {
    background: none;
}

QSlider::add-page:horizontal {
    background: none;
}

QSlider::groove:horizontal {
    background: #333;
    height: 6px;
    border-radius: 3px;
}

QSlider::handle:horizontal {
    background: #aaa;
    border: none;
    width: 12px;
    height: 20px;
    margin: -7px 0;
    border-radius: 6px;
}

/* Ini penting: untuk ticknya */
QSlider::tick-mark:horizontal {
    background: #888;
    width: 1px;
    height: 8px;
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelNIM">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>491</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>F1D022057</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>Font Size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Background Color</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="bgColorSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::sub-page:horizontal {
    background: none;
}

QSlider::add-page:horizontal {
    background: none;
}

QSlider::groove:horizontal {
    background: #333;
    height: 6px;
    border-radius: 3px;
}

QSlider::handle:horizontal {
    background: #aaa;
    border: none;
    width: 12px;
    height: 20px;
    margin: -7px 0;
    border-radius: 6px;
}

/* Ini penting: untuk ticknya */
QSlider::tick-mark:horizontal {
    background: #888;
    width: 1px;
    height: 8px;
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="fontColorSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::sub-page:horizontal {
    background: none;
}

QSlider::add-page:horizontal {
    background: none;
}

QSlider::groove:horizontal {
    background: #333;
    height: 6px;
    border-radius: 3px;
}

QSlider::handle:horizontal {
    background: #aaa;
    border: none;
    width: 12px;
    height: 20px;
    margin: -7px 0;
    border-radius: 6px;
}

/* Ini penting: untuk ticknya */
QSlider::tick-mark:horizontal {
    background: #888;
    width: 1px;
    height: 8px;
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Font Color</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
